documentation/book/con-kafka-mirror-maker-authentication.adoc

```diff
@@ -1,98 +1,55 @@
-// Module included in the following assemblies:
+// This assembly is included in the following assemblies:
 //
-// assembly-kafka-mirror-maker-authentication.adoc
+// assembly-deployment-configuration-kafka-mirror-maker.adoc
 
-[id='con-kafka-mirror-maker-authentication{context}']
-= Authentication support in Kafka Mirror Maker
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
 
-Authentication can be configured in the `KafkaMirrorMaker.spec.consumer.authentication` and `KafkaMirrorMaker.spec.producer.authentication` properties.
-The `authentication` property specifies the type of the authentication method which should be used and additional configuration details depending on the mechanism.
-The currently supported authentication types are:
+[id='assembly-kafka-mirror-maker-authentication-{context}']
 
-* TLS client authentication
-* SASL-based authentication using the SCRAM-SHA-512 mechanism
-* SASL-based authentication using the PLAIN mechanism
+= Authentication
 
-You can use different authentication mechanisms for the Kafka Mirror Maker producer and consumer.
+Use the `consumer.authentication` and `producer.authentication` properties to configure authentication for the consumer and producer.
 
-== TLS Client Authentication
+By default, Kafka Mirror Maker will try to connect to Kafka brokers without any authentication.
+
+Use `authentication.type` to set the authentication mechanism, and add additional configuration details depending on the mechanism.
+
+You can use different authentication mechanisms for the Kafka Mirror Maker consumer and producer.
+
+.TLS Client Authentication
```

Comment:
Could we keep this as a heading? It is quite important because it allows us to link to the specific section.

```suggestion
=== TLS Client Authentication
```

Reply:
I've changed SCRAM Sha and PLAIN to same level too.